packages/react-components/react-charts-preview/src/components/LineChart/LineChart.tsx

```diff
@@ -149,7 +149,7 @@ export const LineChart: React.FunctionComponent<ILineChartProps> = React.forward
     let _createLegendsMemoized: (data: LineChartDataWithIndex[]) => JSX.Element = memoizeFunction(
       (data: LineChartDataWithIndex[]) => _createLegends(data),
     );
-    let _firstRenderOptimization: boolean = false;
```

Comment:
why does this value need to be a ref. It is a constant.

Reply:
Yes, was trying something. Will revert this